diff -rauN mariadb-10.1.32/extra/mariabackup/CMakeLists.txt mariadb-10.1.32-atomic-patch/extra/mariabackup/CMakeLists.txt
--- mariadb-10.1.32/extra/mariabackup/CMakeLists.txt	2018-03-26 16:18:02.000000000 +0200
+++ mariadb-10.1.32-atomic-patch/extra/mariabackup/CMakeLists.txt	2018-03-28 20:44:20.926068353 +0200
@@ -86,7 +86,7 @@
 ADD_SUBDIRECTORY(crc)
 
 
-TARGET_LINK_LIBRARIES(mariabackup sql crc)
+TARGET_LINK_LIBRARIES(mariabackup sql crc atomic)
 
 IF(NOT HAVE_SYSTEM_REGEX)
   TARGET_LINK_LIBRARIES(mariabackup pcreposix)
diff -rauN mariadb-10.1.32/libmysqld/CMakeLists.txt mariadb-10.1.32-atomic-patch/libmysqld/CMakeLists.txt
--- mariadb-10.1.32/libmysqld/CMakeLists.txt	2018-03-26 16:18:02.000000000 +0200
+++ mariadb-10.1.32-atomic-patch/libmysqld/CMakeLists.txt	2018-03-28 20:48:26.807077308 +0200
@@ -184,7 +184,9 @@
     SET_TARGET_PROPERTIES(libmysqld PROPERTIES CLEAN_DIRECT_OUTPUT 1)
     SET_TARGET_PROPERTIES(mysqlserver PROPERTIES CLEAN_DIRECT_OUTPUT 1)
     IF(LIBMYSQLD_SO_EXTRA_LIBS)
-       TARGET_LINK_LIBRARIES(libmysqld ${LIBMYSQLD_SO_EXTRA_LIBS})
+       TARGET_LINK_LIBRARIES(libmysqld ${LIBMYSQLD_SO_EXTRA_LIBS} atomic)
+    ELSE()
+       TARGET_LINK_LIBRARIES(libmysqld atomic)
     ENDIF()
   ENDIF()
 ENDIF()
diff -rauN mariadb-10.1.32/libmysqld/examples/CMakeLists.txt mariadb-10.1.32-atomic-patch/libmysqld/examples/CMakeLists.txt
--- mariadb-10.1.32/libmysqld/examples/CMakeLists.txt	2018-03-26 16:18:02.000000000 +0200
+++ mariadb-10.1.32-atomic-patch/libmysqld/examples/CMakeLists.txt	2018-03-28 20:49:49.911092297 +0200
@@ -27,14 +27,14 @@
 MYSQL_ADD_EXECUTABLE(mysql_embedded ../../client/completion_hash.cc
                ../../client/mysql.cc ../../client/readline.cc
                COMPONENT Client)
-TARGET_LINK_LIBRARIES(mysql_embedded mysqlserver)
+TARGET_LINK_LIBRARIES(mysql_embedded mysqlserver atomic)
 IF(UNIX)
   TARGET_LINK_LIBRARIES(mysql_embedded ${MY_READLINE_LIBRARY})
 ENDIF(UNIX)
 
 MYSQL_ADD_EXECUTABLE(mysqltest_embedded ../../client/mysqltest.cc
   COMPONENT Test)
-TARGET_LINK_LIBRARIES(mysqltest_embedded mysqlserver pcre pcreposix)
+TARGET_LINK_LIBRARIES(mysqltest_embedded mysqlserver pcre pcreposix atomic)
 
 IF(CMAKE_GENERATOR MATCHES "Xcode")
 # It does not seem possible to tell Xcode the resulting target might need
@@ -53,7 +53,7 @@
     COMPONENT Test)
   SET_TARGET_PROPERTIES(mysql_client_test_embedded  PROPERTIES HAS_CXX TRUE)
 ENDIF()
-TARGET_LINK_LIBRARIES(mysql_client_test_embedded mysqlserver)
+TARGET_LINK_LIBRARIES(mysql_client_test_embedded mysqlserver atomic)
 
 IF(UNIX)
 SET_TARGET_PROPERTIES(mysql_embedded PROPERTIES ENABLE_EXPORTS TRUE)
diff -rauN mariadb-10.1.32/sql/CMakeLists.txt mariadb-10.1.32-atomic-patch/sql/CMakeLists.txt
--- mariadb-10.1.32/sql/CMakeLists.txt	2018-03-26 16:18:03.000000000 +0200
+++ mariadb-10.1.32-atomic-patch/sql/CMakeLists.txt	2018-03-28 20:47:43.783551473 +0200
@@ -276,7 +276,7 @@
   ENDIF()
 ENDIF(NOT WITHOUT_DYNAMIC_PLUGINS)
 
-TARGET_LINK_LIBRARIES(mysqld sql)
+TARGET_LINK_LIBRARIES(mysqld sql atomic)
 
 # Provide plugins with minimal set of libraries
 SET(INTERFACE_LIBS ${LIBRT})
diff -rauN mariadb-10.1.32/unittest/sql/CMakeLists.txt mariadb-10.1.32-atomic-patch/unittest/sql/CMakeLists.txt
--- mariadb-10.1.32/unittest/sql/CMakeLists.txt	2018-03-26 16:18:05.000000000 +0200
+++ mariadb-10.1.32-atomic-patch/unittest/sql/CMakeLists.txt	2018-03-28 20:45:33.566958289 +0200
@@ -27,7 +27,7 @@
   ADD_EXECUTABLE(explain_filename-t explain_filename-t.cc)
 ENDIF()
 
-TARGET_LINK_LIBRARIES(explain_filename-t sql mytap)
+TARGET_LINK_LIBRARIES(explain_filename-t sql mytap atomic)
 MY_ADD_TEST(explain_filename)
 
 ADD_EXECUTABLE(mf_iocache-t mf_iocache-t.cc ../../sql/mf_iocache_encr.cc)
